train.py

```diff
@@ -96,6 +98,7 @@ def get_block_class_from_model(model: torch.nn.Module, block_class_name: str) ->
 cramming_offload_gradients_cpu = True
 cramming_activation_checkpointing = True
 cramming_fuse_optim_backward = False
+cramming_use_slimadamw = False
```

Comment:
let's import slimadamw only if `cramming_use_slimadamw` is true

Reply:
and we can also add a comment that `cramming_use_slimadamw` only works when `cramming_offload_optim_cpu` is false